### PR TITLE
fix(grid): Support title overflow not concatenating(#944)

### PR DESCRIPTION
**Pull request by cgfarmer4**
https://github.com/material-components/material-components-web/issues/944#issuecomment-315866959

### CODE REVIEW

**Comment by cgfarmer4** [comment]
I updated the title one to include the property as well instead of being redundant.

File: packages/mdc-grid-list/mdc-grid-list.scss [hunk 1]
```diff
@@ -171,14 +171,13 @@ $mdc-grid-list-tile-secondary-icon-size: 24px;
     font-size: 1rem;
     font-weight: #{map-get($mdc-typography-font-weight-values, medium)};
     line-height: 1rem;
-    text-overflow: ellipsis;
-    white-space: nowrap;
-    overflow: hidden;
+@include mdc-typography-overflow-ellipsis;
```
Comment (touficbatache) [inline packages/mdc-grid-list/mdc-grid-list.scss]: 1- Add an empty line before this one with 0 spaces/indentation
2- Add an indentation of 4 spaces here

File: packages/mdc-grid-list/mdc-grid-list.scss [hunk 2]
```diff
@@ -171,14 +171,13 @@ $mdc-grid-list-tile-secondary-icon-size: 24px;
...
   }
 
   &__support-text {
     @include mdc-typography(body1);
-
+    @include mdc-typography-overflow-ellipsis;
```
Comment (touficbatache) [inline packages/mdc-grid-list/mdc-grid-list.scss]: 3- Remove the empty whitespace at the end

**Comment by cgfarmer4** [comment]
Looks like there are a ton of different constraints to the SCSS that I feel would be necessary to learn to make a concise contribution to this repo. While I would like to contribute, the mental overhead of a pr is too large for the small win. I will continue pointing out bugs and let you guys fix them if you so desire. Thanks again for the awesome repo.

File: packages/mdc-grid-list/mdc-grid-list.scss [hunk 3]
```diff
@@ -171,14 +171,13 @@ $mdc-grid-list-tile-secondary-icon-size: 24px;
...
+
```
Comment (touficbatache) [inline packages/mdc-grid-list/mdc-grid-list.scss]: 4- This shouldn't have an indentation

**Comment by touficbatache** [comment]
Why did you delete this PR?

**Comment by cgfarmer4** [comment]
Please see aforementioned comment. I've spent 1.5 hours on this PR for a fix that took 10 seconds in my own code. I do not wish to spend any more time.

**Comment by touficbatache** [comment]
Ok no problem. I submitted a PR

**Comment by touficbatache** [comment]
Take a look at it and see what you did wrong. You'll do better next time :)

**Comment by cgfarmer4** [comment]
I thought I did OK. There were just a ton of constraints to the formatting that I was unaware of. I'll give it another try next time. Thanks for your support 👍

**Pull request closed without merging**